scss/datetime/_layout.scss

```diff
@@ -1,4 +1,25 @@
+// Infinite calendar
+@mixin hide-scrollbar($dir: 'right') {
```

Comment:
We have the same code in `calendar/_layout.scss`. Extract it to `mixins/core/` and reuse it, to reduce duplication. If the different `$max-scrollbar` value is needed, use a mixin parameter.